lambda/loggingFetch.ts

```diff
@@ -4,7 +4,10 @@ import type { Logger } from '@aws-lambda-powertools/logger'
 
 export const loggingFetch =
 	({ track, log }: { track: AddMetricsFn; log: Logger }) =>
-	async (url: URL, init?: RequestInit): ReturnType<typeof fetch> => {
+	async (
+		url: URL | RequestInfo,
```

Comment:
This is required to allow `loggingFetch` to be able to pass as `fetchImplementation`

Reply:
I've addressed this in a refactor.